samples/dashboards/demo/mqtt/demo.html

```diff
@@ -0,0 +1,75 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/maps/modules/map.js"></script>
+<script src="https://code.highcharts.com/maps/modules/tiledwebmap.js"></script>
+<script src="https://code.highcharts.com/modules/solid-gauge.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/paho-mqtt/1.0.1/mqttws31.js" type="text/javascript"></script>
+
+<link rel="shortcut icon"
+    href="https://wp-assets.highcharts.com/www-highcharts-com/blog/wp-content/uploads/2021/05/19085042/favicon-1.ico">
+
+<!-- Dashboards container -->
+<div id="container">
+    <!-- Top row, control and status bar -->
+    <div class="row">
+        <div class="cell" id="connect-bar">
+            <h1 id="logo-text">Sognekraft</h1>
+            <!--
+            <img id="logo-img-1" src="https://assets.highcharts.com/demos/samples/graphics/dashboards/mqtt/sognekraft_logo.png">
+            -->
+            <!-- Dropdown for selecting power station -->
+            <div id="dropdown-container">
+                <button id="dropdown-button"></button>
+                <div id="dropdown-content" class="dropdown-content">
+                    <!-- Populated dynamically, e.g 
+						<a class="dropdown-select" href="#">Kråkeelvi</a>
+					-->
+                </div>
+            </div>
+            <span id="connect-status">Not connected</span>
+            <div id="connect-switch">
+                <input id="connect-toggle" type="checkbox"></input>
```

Comment:
I would add a label explaining what the toggle does- without reading the text is difficult to understand.
Label above the toggle saying `Connect to mqtt` should be enough.